tests/test_cli.py

```diff
@@ -49,16 +68,14 @@ def start_process(command, is_shell):
                                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     output, error = process.communicate()
     if process.returncode == 0:
-        print('Process Output Message:\n' + str(output) + '\n')
+        return str(output)
     else:
-        command = command.replace(VALID_IOTHUBCONNECTIONSTRING, '******')
+        if VALID_IOTHUBCONNECTIONSTRING in command:
```

Comment:
if the value not in command, nothing should happen. So why need add this check?

Reply:
sometimes if it failed to get this value, and then later all unrelated cases will fail. It is just a security check.

Reply:
Updated code